inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -8,14 +8,38 @@
 #include "functional_test_utils/skip_tests_config.hpp"
 
 std::vector<std::string> disabledTestPatterns() {
-    return {
-            // Issue 26268
-            ".*ConcatLayerTest.*axis=0.*",
-            // Not supported activation types
-            ".*ActivationLayerTest\\.CompareWithRefs/Tanh.*netPRC=FP32.*",
-            ".*ActivationLayerTest\\.CompareWithRefs/Exp.*netPRC=FP32.*",
-            ".*ActivationLayerTest\\.CompareWithRefs/Log.*netPRC=FP32.*",
-            ".*ActivationLayerTest\\.CompareWithRefs/Sigmoid.*netPRC=FP32.*",
-            ".*ActivationLayerTest\\.CompareWithRefs/Relu.*netPRC=FP32.*",
-    };
+    #if defined(_WIN32) || defined(WIN32)
+        return {
+                // Issue 26268
+                ".*ConcatLayerTest.*axis=0.*",
+                // Not supported activation types
+                ".*ActivationLayerTest\\.CompareWithRefs/Tanh.*netPRC=FP32.*",
+                ".*ActivationLayerTest\\.CompareWithRefs/Exp.*netPRC=FP32.*",
+                ".*ActivationLayerTest\\.CompareWithRefs/Log.*netPRC=FP32.*",
+                ".*ActivationLayerTest\\.CompareWithRefs/Sigmoid.*netPRC=FP32.*",
+                ".*ActivationLayerTest\\.CompareWithRefs/Relu.*netPRC=FP32.*",
+                // Issue 31197
+                ".*myriadLayersTestsProposal_smoke\\.Caffe.*",
+                ".*myriadLayersTestsProposal_smoke\\.CaffeNoClipBeforeNms.*",
+                ".*myriadLayersTestsProposal_smoke\\.CaffeClipAfterNms.*",
+                ".*myriadLayersTestsProposal_smoke\\.CaffeNormalizedOutput.*",
+                ".*myriadLayersTestsProposal_smoke\\.TensorFlow.*",
+                ".*IEClassBasicTestP\\.smoke_registerPluginsXMLUnicodePath/0.*",
+                ".*myriadCTCDecoderLayerTests_smoke\\.CTCGreedyDecoder/0.*",
+                ".*myriadCTCDecoderLayerTests_smoke\\.CTCGreedyDecoder/1.*",
+                ".*myriadCTCDecoderLayerTests_smoke\\.CTCGreedyDecoder/2.*",
+                ".*myriadCTCDecoderLayerTests_smoke\\.CTCGreedyDecoder/3.*"
+        };
+    #else
+        return {
+                // Issue 26268
```

Comment:
Looks like there are duplicates in both branches?
I think it's reasonable to instantiate an object (with descriptive name) containing only tests names related to specific issue.